cabal-install/tests/IntegrationTests/internal-libs/new_build.sh

```diff
@@ -1,3 +1,4 @@
 . ./common.sh
 
-cabal new-build p
+cabal new-build p || exit 0
+exit 1 # expect broken
```

Comment:
Just temporary right? So we can expect this to be re-enabled once we have per-component builds.